__tests__/app.test.js

```diff
@@ -408,6 +408,23 @@ describe('DELETE /api/comments/:comment_id',()=>{
         expect(body.message).toBe('Bad Request')
       })
   });
-
 })
 
+describe('GET /api/articles/:article_id (comment_count)',()=>{
+  test("return 200 and article with comment_count 2", () => {
+    return request(app)
+      .get("/api/articles/9")
+      .expect(200)
+      .then(({body})=>{
```

Comment:
Article should come back in an object on a key of 'article' instead of in an array.